packages/wrangler/src/__tests__/workflows.test.ts

```diff
@@ -0,0 +1,434 @@
+  wrangler workflows describe <name>          Describe Workflow resource [open-beta]
+  wrangler workflows trigger <name> [params]  Trigger a workflow, creating a new instance. Can optionally take a JSON string to pass a parameter into the workflow instance [open-beta]
+  wrangler workflows instances                Manage Workflow instances [open-beta]
+
+GLOBAL FLAGS
+  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
+  -c, --config                    Path to .toml configuration file  [string]
+  -e, --env                       Environment to use for operations and .env files  [string]
+  -h, --help                      Show help  [boolean]
+  -v, --version                   Show version number  [boolean]"`
+			);
+		});
+	});
+
+	describe("instances help", () => {
+		it("should show instance help when no argument is passed", async () => {
+			writeWranglerToml();
+
+			await runWrangler(`workflows instances`);
+			await endEventLoop();
+
+			expect(std.out).toMatchInlineSnapshot(
+				`"wrangler workflows instances
+
+Manage Workflow instances [open-beta]
+
+COMMANDS
+  wrangler workflows instances list <name>            Instance related commands (list, describe, terminate, pause, resume) [open-beta]
+  wrangler workflows instances describe <name> <id>   Describe a workflow instance - see its logs, retries and errors [open-beta]
+  wrangler workflows instances terminate <name> <id>  Terminate a workflow instance [open-beta]
+  wrangler workflows instances pause <name> <id>      Pause a workflow instance [open-beta]
+  wrangler workflows instances resume <name> <id>     Resume a workflow instance [open-beta]
+
+GLOBAL FLAGS
+  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
+  -c, --config                    Path to .toml configuration file  [string]
+  -e, --env                       Environment to use for operations and .env files  [string]
+  -h, --help                      Show help  [boolean]
+  -v, --version                   Show version number  [boolean]"`
+			);
+		});
+	});
+
+	describe("list", () => {
+		const mockWorkflows: Workflow[] = [
+			{
+				class_name: "wf_class_1",
+				created_on: "2021-01-01T00:00:00Z",
+				id: "wf_id_1",
+				modified_on: "2021-01-01T00:00:00Z",
+				name: "wf_1",
+				script_name: "wf_script_1",
+			},
+			{
+				class_name: "wf_class_2",
+				created_on: "2022-01-01T00:00:00Z",
+				id: "wf_id_2",
+				modified_on: "2022-01-01T00:00:00Z",
+				name: "wf_2",
+				script_name: "wf_script_2",
+			},
+		];
+
+		const mockGetWorkflows = async (workflows: Workflow[]) => {
+			msw.use(
+				http.get(
+					`*/accounts/:accountId/workflows`,
+					async () => {
+						return HttpResponse.json({
+							success: true,
+							errors: [],
+							messages: [],
+							result: workflows,
+						});
+					},
+					{ once: true }
+				)
+			);
+		};
+
+		it("should get the list of workflows", async () => {
+			writeWranglerToml();
```

Comment:
there's no reason for people to set up a workflow without configuring a binding in wrangler.toml right?  should we be erroring (or warning) if people are running workflows commands without workflows configured? (not necessarily for this PR, but just checking)

Reply:
Yep, there's no reason for people to set up a workflow without configuring a binding in wrangler.toml.